packages/ui/Layout/Layout.js

```diff
@@ -179,7 +179,27 @@ const LayoutBase = kind({
 		 * This includes support for `align-parts` which is shorthand for combining `align-items`
```

Comment:
This includes support shorthand for combining

Reply:
OK. I will fix it.